### DISCUSSION TITLE
Huge emoji icons with Noto Color Emoji font

### DISCUSSION THREAD

**Discussion by manwe-pl** [davatorium/rofi#1940] [(Possible) Issues] [answered]
Hi. Struggling with size of emojis in rofi. Any idea would be appreciated. Without any config (~/.config/rofi) or theme, pure rofi. 

`echo "🙂" | rofi -dmenu`
![2024-01-28-233525_3840x1080_scrot](https://github.com/davatorium/rofi/assets/8430923/b0c52316-8d0c-40e1-b91d-5fe9ed9f3f90)

If I remove Noto Color Emoji config from fonts.conf
```
  <alias binding="strong">
    <family>emoji</family>
    <default><family>Noto Color Emoji</family></default>
  </alias>
```
rofi looks like this
![2024-01-28-233552_3840x1080_scrot](https://github.com/davatorium/rofi/assets/8430923/c4cbab91-a0bf-437e-82ec-44c39574654a)

Playing with rofi's theme and config for a while now and so far nothing is able to shrink that huge smiley face, making https://github.com/fdw/rofimoji unusable.

![2024-01-28-234106_3840x1080_scrot](https://github.com/davatorium/rofi/assets/8430923/40abe751-9a19-4ba4-bb88-4d86ac70b500)

**Comment by DaveDavenport** [comment]
font handling is done via the pango library, rofi is not really involved into the sizing of emojis here.

It also seems to work fine for me:
![rofi-2024-01-29-0850-00000](https://github.com/davatorium/rofi/assets/406245/f7e8cf91-67d2-4e00-9b12-85f69fcb2891)

I am not sure why this happens for you.

**Reply by DaveDavenport** [reply]
https://askubuntu.com/questions/940899/noto-color-emojis-look-super-huge ?

**Reply by manwe-pl** [reply, marked as answer]
Thank you! Works like a charm. For future references, symlink to `/etc/fonts/conf.avail/10-scale-bitmap-fonts.conf` needs to be placed in `/etc/fonts/conf.d`. 
![2024-01-29-104043_3840x1080_scrot](https://github.com/davatorium/rofi/assets/8430923/c8734a2f-47de-43fb-bb0c-7bab62743330)
